examples/apps/attributable-map/tsconfig.json

```diff
@@ -4,7 +4,7 @@
 		"module": "esnext",
 		"outDir": "lib",
 		"jsx": "react",
-		"types": ["jest", "puppeteer", "jest-environment-puppeteer", "expect-puppeteer"],
+		"types": ["jest", "puppeteer", "jest-environment-puppeteer"],
```

Comment:
"puppeteer" can also be removed from this list now, right?

Reply:
The answer is "sometimes". Tests without using the types impacted won't need it, but good to have for consistent, 
 common setup.